core/src/edu/asu/c3simulator/screens/CompanyPanel.java

```diff
@@ -240,45 +229,45 @@ public void render(float delta)
 		stage.act(Gdx.graphics.getDeltaTime());
 		stage.draw();
 	}
-	
+
 	@Override
 	public void resize(int width, int height)
 	{
 		stage.getViewport().update(width, height);
 	}
-	
+
 	@Override
 	public void show()
 	{
 		Gdx.input.setInputProcessor(stage);
-		
+
 	}
-	
+
 	@Override
 	public void hide()
 	{
 		Gdx.input.setInputProcessor(null);
-		
+
 	}
-	
+
 	@Override
 	public void pause()
 	{
 		// do nothing
 	}
-	
+
 	@Override
 	public void resume()
 	{
 		// do nothing
 	}
-	
+
 	@Override
 	public void dispose()
 	{
 		stage.dispose();
 		this.game = null;
-		
+
 	}
-	
+
 }
```

Comment:
I just realized that these spacing changes are reversed. Don't remove indents in the future.